lib/puppet/provider/mount/parsed.rb

```diff
@@ -240,6 +244,15 @@ def self.prefetch(resources = nil)
         end
       end
     end
+    # Examine the catalog to determine whether the fstab should be ordered.
+    # Default is no sorting
+    @sort_output = false
```

Comment:
This seems odd, but I don't see any consistent patterns about using the resources metatype.